smk-edit/static/components/edit-item-details-layer-esri-dynamic.js

```diff
@@ -4,6 +4,47 @@ export default importComponents( [
 ] ).then( function () {
     return vueComponent( import.meta.url, {
         props: [ 'itemId' ],
+        computed: {
+            hasLabels: function() {
+                const layer = this.$store.getters.configLayer( this.itemId );
+                if (layer.dynamicLayers) {
+                    for (const dynamicLayer of layer.dynamicLayers) {
+                        let dynamicJson = JSON.parse(dynamicLayer);
+                        if(dynamicJson.drawingInfo.labelingInfo) {
+                            return true;
+                        }
+                    };
+                }
+                return false;
+            },
+            showLabels: {
+                get: function () {
+                    const layer = this.$store.getters.configLayer( this.itemId );
+                    if (layer.dynamicLayers) {
+                        for (const dynamicLayer of layer.dynamicLayers) {
+                            const dynamicJson = JSON.parse(dynamicLayer);
+                            if(dynamicJson.drawingInfo.labelingInfo && dynamicJson.drawingInfo.showLabels) {
+                                return true;
+                            }
+                        }
+                    }
+                    return false;
+                },
+                set: function ( val ) {
+                    let layer = this.$store.getters.configLayer( this.itemId );
+                    if (layer.dynamicLayers) {
+                        for (let i = 0; i < layer.dynamicLayers.length; i++) {
+                            const dynamicJson = JSON.parse(layer.dynamicLayers[i]);
+                            if(dynamicJson.drawingInfo.labelingInfo) {
```

Comment:
Same here, need a check to make sure `dynamicJson.drawingInfo` is not undefined.